addons/zylann.hterrain/hterrain_data.gd

```diff
@@ -1426,7 +1426,72 @@ func _import_heightmap(fpath: String, min_y: int, max_y: int, big_endian: bool)
 				f.get_16()
 
 		im.unlock()
+	elif ext == "xyz":
+		_logger.debug("detected xyz file")
+		var f := File.new()
+		var err := f.open(fpath, File.READ)
+		if err != OK:
+			return false
+
+		var line = f.get_line()
+		# f.get_8() get byte to string if \r then win ending \r\n (2) else linux ending \n (1)
+		var file_len = f.get_len()
+		var file_res = Util.integer_square_root(file_len / (line.length() + 2)) # + 2 ist /r/n should be dynamic
+		var res := get_adjusted_map_size(file_res, file_res)
+
+		var width := res
+		var height := res
+		var act_min := 10000
```

Comment:
I needed a large number which I could push down with a min operation, since the data will never be higher, than 8,848m, what the mount everest is I placed it at 10k, it could also be integer.max or something.